homeassistant/components/rest_command.py

```diff
@@ -38,6 +38,7 @@
     vol.Required(CONF_URL): cv.template,
     vol.Optional(CONF_METHOD, default=DEFAULT_METHOD):
         vol.All(vol.Lower, vol.In(SUPPORT_REST_METHODS)),
+    vol.Optional(CONF_HEADERS): {cv.string: cv.string},
```

Comment:
This might have to be 
```
vol.Optional(CONF_HEADERS): vol.Schema({cv.string: cv.string})
```

Reply:
I copied that from the rest sensor here: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/rest.py#L38 

Obviously happy to change it if that's the correct way to do things though.

Reply:
Please also fix the `rest_sensor` in this PR. 

I'm 99% sure that is the correct way. @MartinHjelmare can maybe comment too

Reply:
I made that schema change in the two files and rebased this pull request's commits to leave a clean log history when merged.